mypy.ini

```
@@ -218,6 +218,33 @@ ignore_errors = True
 [mypy-torch.overrides]
 ignore_errors = True
 
+[mypy-torch.optim._multi_tensor]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.adam]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.adamw]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.sgd]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.rmsprop]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.rprop]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.asgd]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.adamax]
+ignore_errors = True
+
+[mypy-torch.optim._multi_tensor.adadelta]
+ignore_errors = True
```

The typing changes look fine, but why do we ignore errors here?